packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -265,23 +268,67 @@ class StatefulGraphChatClient implements StatefulClient<GraphChatClient> {
   };
 
   private readonly onActiveAccountChanged = (e: ActiveAccountChanged) => {
-    this.updateUserInfo();
+    if (this.userId !== e.detail.id) {
```

Comment:
That seems to work great! I was able to switch from one account to the next and everything seems very snappy. I was also using a chatId that was valid for both users and it reloaded it magically! Nice stuff!